projects/packages/forms/src/blocks/contact-form/components/jetpack-integrations-modal.js

```diff
@@ -0,0 +1,177 @@
+												__(
+													'Your forms are protected from spam with <a>Akismet</a>!',
+													'jetpack-forms'
+												),
+												{
+													a: (
+														<ExternalLink href={ getRedirectUrl( 'akismet-jetpack-forms-docs' ) } />
+													),
+												}
+											) }
+										</p>
+										<div style={ { display: 'flex', gap: '8px', justifyContent: 'flex-start' } }>
+											<Button
+												variant="secondary"
+												href={ adminUrl }
+												target="_blank"
+												rel="noopener noreferrer"
+												__next40pxDefaultSize={ true }
+											>
+												{ __( 'View spam', 'jetpack-forms' ) }
+											</Button>
+											<Button
+												variant="secondary"
+												href={ akismetUrl }
+												target="_blank"
+												rel="noopener noreferrer"
+												__next40pxDefaultSize={ true }
+											>
+												{ __( 'View stats', 'jetpack-forms' ) }
+											</Button>
+										</div>
+									</>
+								) : (
+									<>
+										<p>
+											{ createInterpolateElement(
+												__(
+													'Akismet is active! There is one step left. Please add your <a>Akismet key</a>.',
+													'jetpack-forms'
+												),
+												{
+													a: <ExternalLink href={ akismetUrl } />,
+												}
+											) }
+										</p>
+										<Button
+											variant="secondary"
+											href={ akismetUrl }
+											target="_blank"
+											rel="noopener noreferrer"
+											__next40pxDefaultSize={ true }
+										>
+											{ __( 'Add Akismet key', 'jetpack-forms' ) }
+										</Button>
+									</>
+								) }
+							</PluginIntegrationPanel>
+						</CardBody>
+					) }
+				</Card>
+
+				<Card>
+					<CardHeader onClick={ () => toggleCard( 'crm' ) } style={ { cursor: 'pointer' } }>
+						<div style={ { display: 'flex', alignItems: 'center', gap: '8px' } }>
+							<Icon icon={ expandedCards.crm ? 'arrow-down-alt2' : 'arrow-right-alt2' } />
+							<strong>{ __( 'Jetpack CRM', 'jetpack-forms' ) }</strong>
+						</div>
+					</CardHeader>
+					{ expandedCards.crm && (
+						<CardBody>
+							<CRMIntegrationSettings
+								jetpackCRM={ attributes.jetpackCRM }
+								setAttributes={ setAttributes }
+							/>
+						</CardBody>
+					) }
+				</Card>
+
+				<Card>
+					<CardHeader
+						onClick={ () => toggleCard( 'creativemail' ) }
+						style={ { cursor: 'pointer' } }
+					>
+						<div style={ { display: 'flex', alignItems: 'center', gap: '8px' } }>
+							<Icon icon={ expandedCards.creativemail ? 'arrow-down-alt2' : 'arrow-right-alt2' } />
+							<strong>{ __( 'Creative Mail', 'jetpack-forms' ) }</strong>
+						</div>
+					</CardHeader>
+					{ expandedCards.creativemail && (
+						<CardBody>
+							<NewsletterIntegrationSettings />
+						</CardBody>
+					) }
+				</Card>
+			</div>
+		</Modal>
+	);
+};
+
+export default JetpackIntegrationsModal;
```

Comment:
This component is going to change dramatically as we build out the modal. I wouldn't spend a lot of time doing detailed review or feedback on this. Think of it as a placeholder. It should work, be sane, and provide a platform for development.